src/styles/actor/party/kingdom/_activities.scss

```diff
@@ -86,6 +86,7 @@
         float: right;
         margin: 0;
         margin-left: auto;
+        margin-top: -0.125rem;
```

Comment:
Might be time to have a single `margin` line

Reply:
It also turns out margin 0 is no longer required due to global tag changes but better safe than sorry.